### PR TITLE
fix(scala-sdk): replicated entities CCE after entity passivation

### PR DESCRIPTION
**Pull request by efgpinto**
Resolves https://github.com/lightbend/kalix-jvm-sdk/issues/1289

### CODE REVIEW

File: samples/scala-replicatedentity-examples/src/test/scala/com/example/replicated/counter/SomeCounterIntegrationSpec.scala [hunk 1]
```diff
@@ -41,6 +41,16 @@ class SomeCounterIntegrationSpec extends AnyWordSpec with Matchers with BeforeAn
...
 
+    "Test value after passivation" in {
+      val counterId = "1"
+
+      counter.increase(IncreaseValue(counterId, 1)).futureValue
+      Thread.sleep(5000)
+
+      val getResult = counter.get(GetValue(counterId))
+      getResult.futureValue.value shouldBe 1
+    }
```
Comment (efgpinto) [inline samples/scala-replicatedentity-examples/src/test/scala/com/example/replicated/counter/SomeCounterIntegrationSpec.scala:52]: I just wanted to have a test somewhere confirming behaviour. I'm not sure we should add one of these for each entity, or just wait for the proper TCK tests to be updated.

**Review by efgpinto — COMMENTED** [review]
So, I think it is better to decouple this from the ongoing TCK work. So opening for review now.

File: samples/scala-replicatedentity-examples/src/test/scala/com/example/replicated/counter/SomeCounterIntegrationSpec.scala (hunk 1 above)
Comment (pvlugter) [reply samples/scala-replicatedentity-examples/src/test/scala/com/example/replicated/counter/SomeCounterIntegrationSpec.scala:52]: Just this for now seems good, remove again once TCK tests added.

**Review by pvlugter — APPROVED** [review]
LGTM. And then follow up with the TCK tests.

File: samples/scala-replicatedentity-examples/src/test/scala/com/example/replicated/counter/SomeCounterIntegrationSpec.scala (hunk 1 above)
Comment (efgpinto) [reply samples/scala-replicatedentity-examples/src/test/scala/com/example/replicated/counter/SomeCounterIntegrationSpec.scala:53]: tks, added a comment to remove after tck tests added.

**Pull request merged**